include/mbedtls/check_config.h

```diff
@@ -23,6 +23,7 @@
 #ifndef MBEDTLS_CHECK_CONFIG_H
 #define MBEDTLS_CHECK_CONFIG_H
 
+/* *INDENT-OFF* */
```

Comment:
Note: this is probably planned already, but I think for the benefit of people who are not already familiar with Uncrustify, it should be mentioned somewhere in our documentation that we can use this for parts of the code we don't want reformatted/checked.